document/core/syntax/instructions.rst

```diff
@@ -235,15 +413,21 @@ Instructions in this group are concerned with linear :ref:`memory <syntax-mem>`.
    \production{instruction} & \instr &::=&
      \dots \\&&|&
      \K{i}\X{nn}\K{.}\LOAD~\memarg ~|~
-     \K{f}\X{nn}\K{.}\LOAD~\memarg \\&&|&
+     \K{f}\X{nn}\K{.}\LOAD~\memarg ~|~
+     \K{v128.}\LOAD~\memarg \\&&|&
      \K{i}\X{nn}\K{.}\STORE~\memarg ~|~
-     \K{f}\X{nn}\K{.}\STORE~\memarg \\&&|&
+     \K{f}\X{nn}\K{.}\STORE~\memarg ~|~
+     \K{v128.}\STORE~\memarg \\&&|&
      \K{i}\X{nn}\K{.}\LOAD\K{8\_}\sx~\memarg ~|~
      \K{i}\X{nn}\K{.}\LOAD\K{16\_}\sx~\memarg ~|~
      \K{i64.}\LOAD\K{32\_}\sx~\memarg \\&&|&
      \K{i}\X{nn}\K{.}\STORE\K{8}~\memarg ~|~
      \K{i}\X{nn}\K{.}\STORE\K{16}~\memarg ~|~
      \K{i64.}\STORE\K{32}~\memarg \\&&|&
+     \K{i16x8.}\LOAD\K{8x8}\_\sx~\memarg ~|~
+     \K{i32x4.}\LOAD\K{16x4}\_\sx~\memarg ~|~
+     \K{i64x2.}\LOAD\K{32x2}\_\sx~\memarg \\&&|&
+     \X{vxx}\K{.}\LOAD\K{\_splat}~\memarg \\&&|&
```

Comment:
This is incorrect. vxx represents ixx and fxx (i8x16... f64x2). It should be `v8x16`, `v16x8`, `v32x4`, `v64x2`, might need another production for here.

Reply:
Is there a chance we can just rename all the `vNxM` prefixes to `iNxM`? I understand the reason is that it doesn't matter that they are ints, but the same is true for, say, plain `i32.and` or `i32.load`.

Reply:
The difference is that `i32.load`  returns `i32` and we have `f32.load` if we want a `f32`, whereas these all return a `v128`. Naming it with a `i` prefix suggests that it is implemented using integer loads, where it is not always the case, and also might cause confusion because there is no `f` counterpart for loading floats.

That said, I don't have a strong preference either way. I can file an issue and see if anyone has comments.
@tlively if you have comments on this re: toolchain or initial rationale for naming loads.

Reply:
I actually would prefer using `i` prefixes for these as well because the `v` prefixes are used on few enough instructions that they seem surprising and because they do not correspond to anything in MVP. I recall [discussing](https://github.com/WebAssembly/simd/pull/82#issuecomment-519893850) this for specific instructions in the past, but the `v` prefix was [present](https://github.com/WebAssembly/simd/pull/1/files#diff-28263aca205221ef5f3c6b4a232654c3R281-R285) in the original proposal writeup, so I don't think we've discussed making this change for all instructions before.

There's no toolchain-specific reason to make this change, though.

Reply:
How much simpler is the definition here if we make the change? I have to admit I'm a bit hesitant to suggest changing the instruction names for the simplicity of the spec document.

Reply:
> How much simpler is the definition here if we make the change?

This definition of load_splats will become `ixx.load_splat`.

If we stick to `v8x16` et al, we will need to:

1. Define `vX`. Right now we rely on the definition of `iN` to describe the *shape* of SIMD, using `vX` means we need to somehow describe `vX`, but in a way that makes it not part of the syntax
2. After `vX` we can talk about `vXxN`

It won't be a lot more text, but it will be very specialized to these 6 instructions, which (like Thomas says) does not have any analogues in MVP.

We *could* define `f32x4.load_splat` and `f64x2.load_splat`, but have them be "aliases", so share the same opcode and semantics as the integer counterparts...

Reply:
I would also be fine just having `i32x4.load_splat` and friends and not having `f32x4.load_splat`. We could also avoid the problem (for these instructions) by renaming them to `v128.load<width>_splat` (and following suit for the load-and-zero instructions).

Reply:
Yea I quite like the `v128.load<width>_splat` notation we were discussing in #237 , then we can side-step this issue for load_splat, and rename swizzle and shuffle to `i8x16`.

Reply:
Filed #297 to discuss this more, and see if anyone supports/objects.

Reply:
#297 hasn't had too many comments outside of the people already involved here, but I think it will be a fine change. The only other change is `v8x16.shuffle` and `v8x16.swizzle`, which I have filed #316 for.

Reply:
I have optimistically made the naming changes (of load splats, load extends, shuffle, swizze) in the latest commit. With that, we no longer have special `v...` prefix, so no need to define them, and loads all start with `v128`.